api/tests/unit/domain/usecases/get-schooling-registrations-csv-template_test.js

```diff
@@ -76,10 +107,24 @@ describe('Unit | UseCase | get-schooling-registrations-csv-template', () => {
     });
   });
 
-  context('When user is ADMIN in an SUP organization but it does not manage students', () => {
+  context('When user is not ADMIN in a SCO-AGRICULTURE organization', () => {
     it('should throw an error', async () => {
       // given
-      sinon.stub(membershipRepository, 'findByUserIdAndOrganizationId').resolves([{ isAdmin: true, organization: { isManagingStudents: false, type: 'SUP' } }]);
+      process.env['AGRICULTURE_ORGANIZATION_ID'] = '1';
+      sinon.stub(membershipRepository, 'findByUserIdAndOrganizationId').resolves([{ isAdmin: false, organization: { id: '1', isManagingStudents: true, type: 'SCO' } }]);
+
+      // when
+      const error = await catchErr(getSchoolingRegistrationsCsvTemplate)({ userId, organizationId, membershipRepository });
+
+      // then
+      expect(error).to.be.instanceOf(UserNotAuthorizedToAccessEntity);
+    });
+  });
+
+  context('When user is not a member of the organization', () => {
```

Comment:
il manque un test `When user is ADMIN in a PRO organization, it should throw an error`,